docs/agent-k8s-install-helm.md

```diff
@@ -0,0 +1,117 @@
+# Install Using Helm
+
+Use the Helm package manager to install the Smart Agent to Kubernetes
+environments.
+
+If you don't have Helm installed already, use kubectl
+to install the Smart Agent. To learn more, see
+[Install the Smart Agent using kubectl](agent-k8s-install-kubectl.md).
+
+## Prerequisites
+
+* Linux kernel version 2.6.32 or higher
+* cap_dac_read_search and cap_sys_ptrace capabilities
+* Terminal or a similar command-line interface application
+* Helm:
+
+  - The installation chart for the Smart Agent is compatible with Helm version 2 or Helm version 3.
+  - To learn more, see the [Helm site](https://helm.sh/).
+
+* Tiller installed on all of your Kubernetes hosts. Because of
+  the way that the installation chart works, you need Tiller even if you're using
+  Helm version 3.
+* SignalFx realm for your organization. See [Realms](../../../_sidebars-and-includes/smart-agent-realm-note.html).
+* A SignalFx access token. See [Smart Agent Access Token](../../../_sidebars-and-includes/smart-agent-access-token.html).
+
+
+## Configure Helm for the Smart Agent
+
+The SignalFx Smart Agent chart for Helm comes with a values.yaml configuration
+file that contains useful default values. If you want to modify these values,
+override them by creating your own values file. Follow these steps:
+
+1. Download the default [values.yaml](https://github.com/signalfx/signalfx-agent/blob/master/deployments/k8s/helm/signalfx-agent/values.yaml) file.
+2. Rename the file. For example, rename it to myValues.yaml.
+3. Edit the file to update the values with your own choices, then save the file.
+
+When you install the Smart Agent using Helm, add the `-f <values_yaml_file>` parameter to the install command.
+For example, add the `-f myValues.yaml` parameter.
+
+## Install with Helm
+
+1. Remove collector services such as `collectd`.
```

Comment:
Have there been any known incompatibilities?  I can't really imagine any scenario in which other agents would cause an issue.

Reply:
The Smart Agent [Quick Install](https://docs.signalfx.com/en/latest/integrations/agent/quick-install.html#quick-install) topic, in the section [General](https://docs.signalfx.com/en/latest/integrations/agent/quick-install.html#general) contains this admonition:

> Uninstall or disable any previously-installed collector agents from your host, such as `collectd`.

What would you like to say here? I can remove the statement if necessary.